### PR TITLE
Refactor Components to Functional Components

### PR DESCRIPTION
**Pull request by BenjaminSemah**
#### In this branch, I have done the following;
- Refactored the `Calculator` component from a class based to a functional component with hooks,
- Tested the app to ensure the functionality still works,
- Fixed errors spotted by linters.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
-}
+// class App extends React.Component {
+//   render() {
+//     return (
+//       <>
+//         <Calculator />
+//       </>
+//     );
+//   }
+// }
```
Comment (leolpaz) [inline src/App.js]: - [x] As pointed in the JavaScript requirements of the project in JavaScript best practice's kindly remove the commented code, you can always check old commits if you want to see how it was before

image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @BenjaminSemah , 

You've done an amazing job so far, I'd like to highlight the following:
- Great job with the refactor
- App still works properly

Some issues need to be resolved before your approval, but you're almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenjaminSemah** [comment]
### Delete Commented Out Code.
- I have deleted the commented out code as pointed out by previous reviewer.

**Review by leolpaz — APPROVED** [review]
Hi @BenjaminSemah , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the changes required by the previous reviewer (well... me)

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**